### PR TITLE
New home page

### PR DESCRIPTION
**Pull request by hannahouazzane**
Fixes #
Added designs for the home page.

### CODE REVIEW

File: src/components/BlueText.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React from 'react'
```
Comment (Nikomus) [inline src/components/BlueText.js:1]: Is this component used? There are no references to it in other files

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        font-weight:700;
+
+        ${margin_left ? `margin-left: ${margin_left};` : ''}
+        font-size:1.25rem;
+        font-family: KeepCalm;
+        font-weight: bold;
+      }
+
+      button:hover {
+        background-color: #48e9ce;
```
Comment (Nikomus) [inline src/components/Button.js:47]: Are there variables for these?

File: src/components/Text.js [hunk 3]
```diff
@@ -20,8 +19,9 @@ const Text = ({
       .text {
         ${weight ? `font-weight: ${weight};` : ''}
         ${fontSizes[size] ? `font-size: ${fontSizes[size]};` : ''}
-        ${lineHeight ? `line-height: ${lineHeight};` : ''}
```
Comment (Nikomus) [inline src/components/Text.js:23]: lineHeight is still in the propTypes and passed in a lot of places, should this be replaced?

File: src/components/Button.js (hunk 2 above)
Comment (rosejbon) [reply src/components/Button.js:47]: I don't think they currently exist in the codebase, but they could be added to `theme.js`, something like
```
const c_NEON_TEAL = '#48e9ce'
```

File: src/components/Smile.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import classNames from 'classnames'
+import arc from '../components/pinkArc.png'
+// TODO: Should we have a block level version which uses a 0 margin p?
```
Comment (rosejbon) [inline src/components/Smile.js:4]: Do we need this comment here? :)

File: src/components/Text.js [hunk 5]
```diff
@@ -4,14 +4,13 @@ import classNames from 'classnames'
 
 import { fontSizes } from '../theme'
 
-// TODO: Should we have a block level version which uses a 0 margin p?
```
Comment (rosejbon) [inline src/components/Text.js:7]: Interestingly I would keep this TODO in as I think it was left by a previous dev I think. I tend to have a rule where I don't delete other devs TODOs unless I've completed them or they are happy for me to.

File: src/components/Text.js [hunk 6]
```diff
@@ -35,10 +35,19 @@ const Text = ({
...
 Text.propTypes = {
   children: PropTypes.node,
-  weight: PropTypes.oneOf([400, 500, 700]),
+  weight: PropTypes.string,
   align: PropTypes.string,
   lineHeight: PropTypes.number,
   size: PropTypes.oneOf(Object.keys(fontSizes)),
+  color: PropTypes.string,
+}
+//default values to avoid any injections?
```
Comment (rosejbon) [inline src/components/Text.js:44]: Was this a note to yourself or a future TODO for yourself or other devs? :)

**Review by rosejbon — APPROVED** [review]

File: src/components/Text.js [hunk 7]
```diff
@@ -35,10 +37,19 @@ const Text = ({
...
   size: PropTypes.oneOf(Object.keys(fontSizes)),
+  color: PropTypes.string,
+}
+
+Text.defaultProps = {
+  weight: '400',
+  align: 'inherit',
+  lineHeight: 'inherit',
+  size: 'inherit',
+  color: 'inheirt',
```
Comment (Nikomus) [inline src/components/Text.js:52]: 🕵️

**Pull request merged**